TypeCobol.Analysis/Dfa/DfaUsePoint.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TypeCobol.Analysis.Dfa
+{
+    /// <summary>
+    /// Generic class representing a USE Point.
+    /// </summary>
+    /// <typeparam name="I">Instruction generic Type</typeparam>
+    /// <typeparam name="V">Used Variable generic Type</typeparam>
+    public class DfaUsePoint<I, V>
```

Comment:
Same remark as for DEF points, I'll leave it as it is now until we decide for the DEF points properties accessibility.

Reply:
Same answer as for DEF.